contract_compassion/wizard/__init__.py

```diff
@@ -0,0 +1,15 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Cyril Sester <csester@compassion.ch>

```

Comment:
I didn't remember to have coded this. Worse, if I blame the code it shows me that DavidZIVI committed this